index.js

```diff
@@ -38,10 +39,16 @@ function Store (dbName, options) {
   var db = new CustomPouchDB(dbName)
   var emitter = new EventEmitter()
   var remote = options.remote
+  var HoodieSync = db.hoodieSync({remote: remote, emitter: emitter})
   var api = merge(
-    db.hoodieSync({remote: remote, emitter: emitter}),
     db.hoodieApi({emitter: emitter}),
-    { hasLocalChanges: hasLocalChanges.bind(db) }
+    HoodieSync,
+    {
+      hasLocalChanges: hasLocalChanges.bind(db),
+      push: syncWrapper.bind(HoodieSync, 'push'),
+      pull: syncWrapper.bind(HoodieSync, 'pull'),
+      sync: syncWrapper.bind(HoodieSync, 'sync')
```

Comment:
I wouldn't use a capitalized `HoodieSync` variable name here, as it's not a "class" but an instance. So I'd rather call it `syncApi` for example. Thoughts?